baseplate/core.py

```diff
@@ -170,7 +183,14 @@ def _secret(self):
         return self._secret_store.get_simple("jwt/authentication/secret")
 
     def attach_context(self, context):
-        """Attach this authentication wrapper to the provided context
+        """DEPRECATED - Attach this authentication wrapper to the given context
```

Comment:
you can use `.. deprecated::` instead: http://www.sphinx-doc.org/en/stable/markup/para.html#directive-deprecated